docker/deb-rpm-s3/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+#
+# Build script for the deb-rpm-s3 docker container.
+#
+
+cd "$(dirname "$0")"
+
+if [ ! -e "elasticsearch.asc" ]; then
+    cat << EOF
+You must place a copy of the Elasticsearch GPG signing key (named
+elasticsearch.asc) into
```

Comment:
Correct, the password protected GPG key will be contained in the docker image. I don't know of any way to blacklist the container from being pushed. But I think the fact that it is not named `username/deb-rpm-s3` should prevent it from being pushed to docker-hub.